code/modules/mob/living/silicon/login.dm

```diff
@@ -1,11 +1,11 @@
 /mob/living/silicon/Login()
 	SetSleeping(0)
 	if(mind && SSticker && SSticker.mode)
+		mind.remove_all_antag_datums()
```

Comment:
won't this cause issues with malf AI?

Reply:
Yeah potential issue here, I'll be looking into it.

Reply:
This was an issue. Reverted the change.